index.html

```diff
@@ -414,7 +414,28 @@
           <div class="col row_abstract">
             در این گفتگو ظرافت‌های تدوین قانون برای مؤسسات مالی فعال در زمینه بیت‌کوین، همچنین وظیفه کاربران بیت‌کوین در راه مقابله با قوانین سخت‌گیرانه به تفصیل مورد بحث قرار می‌گیرند. مطالعه این گفتگو به سیاست‌گذاران، کارآفرینان مؤسسات مالی، و از همه مهم‌تر بیت‌کوینرهای فارسی زبان پیشنهاد می‌شود. حامی ترجمه این پادکست سایت bitcoind.me است.
           </div>
-        </div>        
+        </div>    
+
+        <div class="row pb-2">
+          <div class="col-lg-4 row_itemTitle row_gotoLink">
+            <a href="blobs/books/PasswordBySevil.pdf">
+              پسورد، پسفریز، رمز عبور
+            </a>
+          </div>
+          <div class="col-lg-6 row_itemDesc">
+            نوشته: <a href="https://twitter.com/seviltheme" target="_blank">سویل .ک</a>
+          </div>
+          <div class="col-lg-2 row_downloadLink">
+            <a href="blobs/books/PasswordBySevil.pdf">
```

Comment:
لطفا پس از تغییر مسیر فایل، این مسیر رو هم تصحیح کنید.
```suggestion
            <a href="blobs/tuts/PasswordBySevil.pdf">
```